components/NavBar/FirstPageNavbar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styles from "@/styles/Header.module.css";
+import NavbarLink from "@/components/Navbar/NavbarLink";
```

Comment:
![image](https://user-images.githubusercontent.com/54622899/218695445-2816a770-f2dc-4dd2-995c-3e337e2f73ed.png)
Here you imported NavbarLink from folder "Navbar", however I noticed the folder name in your source code is "NavBar"
![image](https://user-images.githubusercontent.com/54622899/218696029-d91f0850-04a5-475f-a9cd-a77b4f8d4df7.png)
Maybe this caused failed to compile on vercel?

Reply:
thank you ~I forgot this folder name changed

Reply:
I saw you created a "layout" folder to place your Header.tsx while you put NavbarLink and so on in the "components" folder. I totally understand your concern. Maybe all of us should discuss about the file structure on tomorrow meeting to decide:
1. Where to put files that implement each section of our website
2. Where to put sub-components of those files for each section, like your NavbarLink
3. Where to put utilities that might be used at anywhere of the website, like buttons
4. How to structure the corresponding test files for these 3 different types above
I think it would better to discuss this thoroughly to minimize the inconsistency.